addons/website/static/src/xml/website.gallery.xml

```diff
@@ -13,8 +13,8 @@
         <div t-attf-id="#{id}" class="carousel slide" data-ride="carousel" t-attf-data-interval="#{interval}" style="margin: 0 12px;">
             <div class="carousel-inner" style="padding: 0;">
                  <t t-foreach="images" t-as="image">
-                    <div t-attf-class="carousel-item #{image_index == index and 'active' or ''}">
-                        <img t-attf-class="img img-fluid d-block #{userStyle}" t-att-src="image.src" t-att-alt="image.alt"/>
+                    <div t-attf-class="carousel-item #{image_index == index and 'active' or None}">
+                        <img t-attf-class="#{attrClass ? attrClass : 'img img-fluid d-block'}" t-att-src="image.src" t-attf-style="#{attrStyle}" t-att-alt="image.alt" data-name="Image"/>
```

Comment:
Side note: agreed but only because it is a static file :wink: For "python" qweb template, you should not use t-att- for the class attribute otherwise it cannot be modified through xpath correctly (and this is a common attribute we want to xpath/modify)